dinov2/configs/train/whoi.yaml

```diff
@@ -60,21 +60,24 @@ ibot:
   head_hidden_dim: 2048
 train:
   batch_size_per_gpu: 256
-  dataset_path: HDF5Dataset:split=TRAIN:root=/fast/AG_Kainmueller/plankton/data/WHOI/preprocessed_hdf5:extra=*
+  dataset_path: LMDBDataset:split=ALL:root=/fast/AG_Kainmueller/plankton/data/WHOI/preprocessed/preprocessed_lmdb/:extra=*
   output_dir: 'home/jluesch/output_dir'
-  saveckp_freq: 15
+  saveckp_freq: 10
   seed: 0
-  num_workers: 4
-  OFFICIAL_EPOCH_LENGTH: 1250
-  cache_dataset: true
+  num_workers: 8
+  OFFICIAL_EPOCH_LENGTH: 2500
   centering: "sinkhorn_knopp" # or "centering"
+  cache_dataset: false
```

Comment:
why is the dataset not cached?

Reply:
Caching the dataseyt did not show any speed improvements until now, so leaving default (not cached)